src/components/LogoutLink.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Link from 'react-router/lib/Link';
+
+/**
+ * Logout link component
+ * @module LogoutLink
+ */
+class LogoutLink extends React.Component {
+	render() {
+		const {
+			to,
+			children,
+			...other
+		} = this.props;
+
+		return (
+			<Link
+				to={{ pathname: to, query: {logout: true} }}
```

Comment:
That would work. We haven't set up redirects in our React Router yet, but we'll probably want to build _some_ of the logout-and-redirect functionality into the regular routing system, e.g. redirecting to the group homepage if someone tries to access a group admin page that they don't have authorization to access.